composer.json

```diff
@@ -71,6 +71,7 @@
         "psr/http-message": "^1.0",
         "psr/log": "^2.0",
         "ramsey/uuid": "^4.0",
+        "ramsey/uuid-doctrine": "^2.0",
```

Comment:
I think usage of `ramsey/uuid` is being deprecated in favor of `symfony/uid`

Reply:
I did not know about that, I used `ramsey/uuid` for a while. I will investigate on it. Thank you for pointing this out

Reply:
I also switched the AttributeBundle to `symfony/uid` since it just use the V1 generation on the model side and V4 in the tests (don't know why it use a different uuid version...)

Reply:
Eratum, it creates some issues on Behat tests apparently, so I removed my commit. If someone wants to do it into another PR, it seems to be able to remove this dependency.